docs/source/host_challenge.md

```diff
@@ -125,37 +131,106 @@ Finally run the `./run.sh` script in the bundle. It will generate a `challenge_c
+Please ensure the following fields are set to the following values for static code-upload based challenges:
 
-- `remote_evaluation` is set to `True`
-- `is_docker_based` is set to `True`
+- `remote_evaluation : True`
+- `is_docker_based : True`
+- `is_static_dataset_code_upload : True`
 
-For reference to the fields, refer to the [challenge configuration reference section](configuration.html).
+In order to perform evaluation, you might also need to create an EKS cluster on AWS. This is because we use docker containers for managing the evaluation environment, and as well as model container. See [AWS Elastic Kubernetes Service](https://aws.amazon.com/eks/) to learn more about what EKS is and how it works.
 
-### Step 3: Edit evaluation script
+We need the following details for the EKS cluster in order to perform evaluations in case you are using your own AWS account:
 
-Next step is to edit the challenge evaluation script that decides what metrics the submissions are going to be evaluated on for different phases.
+- `aws_account_id: <AWS Account ID>`
+- `aws_access_key_id: <AWS Access Key ID>`
+- `aws_secret_access_key: <AWS Secret Access Key>`
+- `aws_region: <AWS Region>`
 
-Please refer to the [writing evaluation script](evaluation_scripts.html) to complete this step.
+These details need to be added to the challenge configuration.
 
-### Step 4: Edit challenge HTML templates
+For reference to the other fields, refer to the [challenge configuration reference section](configuration.html).
+
+### Step 3: Save the static dataset using EFS
+
+Use [AWS EFS](https://aws.amazon.com/efs/) to store the static dataset file(s) on which the evaluation is to be performed. By default an `EFS` file system is created and the file system ID is stored in `efs_id` in the Challenge Evaluation Cluster and then the file system is mounted on the instances inside the cluster.
+
+### Step 4: Create an example docker image for submission
+
+Create a Dockerfile showing the participants how to install their requirements and run the submission script which produces the predictions file. This docker image is then run on the cluster to perform predictions.
+
+An template Dockerfile is shown below:
+
+```Dockerfile
+FROM nvidia/cuda:11.2.0-cudnn8-runtime-ubuntu20.04
+
+RUN apt-get update &&\
+    DEBIAN_FRONTEND=noninteractive apt-get install -y python3 &&\
+    apt-get install -y
+
+# ADITIONAL PYTHON DEPENDENCIES
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+
+WORKDIR /app
+
+# COPY WHATEVER OTHER SCRIPTS YOU MAY NEED
+COPY trained_model /trained_model
+# SPECIFY THE ENTRYPOINT SCRIPT
+CMD ["python", "-u", "submission.py"]
+```
+
+This docker image will run `submission.py` script and the script will save the predictions at the specified location.
+
+### Step 5: Edit evaluation script
+
+Next step is to create static code-upload challenge evaluation that decides what metrics the submissions are going to be evaluated on for different phases.
```

Comment:
`Next step is to write the evaluation script to compute the metrics for each submission`